src/components/MetricsPane/MetricsPane.jsx

```diff
@@ -279,63 +288,73 @@ const MetricsPane = ({ showMetricsPane, setShowMetricsPane, view }) => {
     )
   }
 
-  const RenderSummarizedMetrics = () => {
-    return (
-      <SummarizedMetrics
-        onClick={toggleMobileMetricsPane}
-        $isDesktopWidth={isDesktopWidth}
-        $showMobileExpandedMetricsPane={showMobileExpandedMetricsPane}
-      >
-        <MetricsCard>
-          <P>{getActiveProjectCount()}</P>
-          <H3>Projects </H3>
-        </MetricsCard>
-        <SurveysAndTransectsContainer>
-          <MetricsCard>
-            <P>{numSurveys}</P>
-            <H3>Surveys</H3>
-          </MetricsCard>
-          <MetricsCard>
-            <P>{numTransects}</P>
-            <H3>Transects</H3>
-          </MetricsCard>
-        </SurveysAndTransectsContainer>
+  const countryLabel = numUniqueCountries === 1 ? 'Country' : 'Countries'
+
+  const transectAndProjectCountCards = (
+    <>
+      <MetricsCard>
+        <MetricCardPMedium>{numTransects.toLocaleString()}</MetricCardPMedium>
+        <MetricCardH3>Transects</MetricCardH3>
+      </MetricsCard>
+      <MetricsCard>
+        <MetricCardPMedium>{getActiveProjectCount().toLocaleString()}</MetricCardPMedium>
+        <MetricCardH3>Projects </MetricCardH3>
+      </MetricsCard>
+    </>
+  )
+
+  const displayedProjectsMetrics = (
+    <SummarizedMetrics
+      onClick={toggleMobileMetricsPane}
+      $isDesktopWidth={isDesktopWidth}
+      $showMobileExpandedMetricsPane={showMobileExpandedMetricsPane}
+      $showLoadingIndicator={showLoadingIndicator}
+    >
+      <MetricsCard>
+        <MetricCardPBig>{numSurveys.toLocaleString()}</MetricCardPBig>
+        <MetricCardH3>Surveys</MetricCardH3>
+      </MetricsCard>
+      <MetricsCard>
         {isDesktopWidth ? (
```

Comment:
If we had the luxury of more time and I was a CSS guru, I would have liked to not need a JS conditional for `isDesktopWidth` and have separate markup for desktop and mobile, but that also might be overly idealistic (its been a while since I did anything truly responsive TBH). Aligning inline elements next to block elements in CSS is finicky, so I opted in mobile view just to use the same block elements as the other metric cards and be done with it.